WalletWasabi.Standard/CoinJoin/Common/Models/RoundStateResponseBase.cs

```diff
@@ -22,6 +22,8 @@ public abstract class RoundStateResponseBase
 
 		public int RegisteredPeerCount { get; set; }
 
+		public int UnconfirmedPeerCount { get; set; }
```

Comment:
Do you think Unconfirmed is the best name? I think this can get conflated with peers whose funds come from unconfirmed transactions. perhaps QueuedPeers as Queued is directly >= Registered?

Reply:
Can users who unconfirmed, re-confirm their status in the round? If so I think QueuedPeers is an appropriate name

Reply:
you mean un-registered?

Reply:
yes to me unconfirmed and un-register are synonymous